templates/download/server/arm.html

```diff
@@ -101,9 +101,9 @@ <h2>
         <li class="p-list__item is-ticked">
           Application container technology based on Docker and Kubernetes, including FAN-based networking.
         </li>
-        <li class="p-list__item is-ticked">SQL and NoSQL databases including CouchDB, MySQL, PostgreSQL, Redis and SQLite.</li>
+        <li class="p-list__item is-ticked">SQL and NoSQL databases including MySQL, PostgreSQL, Valkey and SQLite.</li>
         <li class="p-list__item is-ticked">
-          A wide collection of web technologies, from HTTP servers to frameworks including Apache, Django, memcached, Nginx, WordPress and Varnish.
+          A wide collection of web technologies, from HTTP servers to frameworks including Apache, Nginx, Django, memcached and Varnish.
         </li>
       </ul>
     </div>
```

Comment:
Reviewing this made me spot something that was missed in the copydoc.
A few lines below we have "Built for cutting-edge hardware, from the HP Moonshot range to standard form-factor certified systems, Ubuntu and ARM Server provide truly compelling economics for cloud-scale deployment."

This is stupid old statement that aged badly, HPE Moonshot was discontinued 3 years ago.

Suggesting a more product neutral statement
"Built for cutting-edge platforms, from cloud instances to arm servers and standard form-factor systems, Ubuntu and ARM Server provide truly compelling economics for scale deployments at scale."